docs/usage/node.md

```diff
@@ -17,45 +17,6 @@ Renovate can manage the Node.js version in the following files:
 - The [`.nvmrc`](https://github.com/creationix/nvm#nvmrc) file for the [Node Version Manager](https://github.com/creationix/nvm)
 - The [`node_js`](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions) field in [`.travis.yml`](https://docs.travis-ci.com/user/customizing-the-build/)
 
-## How It Works
-
-Node.js renovation in `package.json > engines` and in `.nvmrc` is enabled by default, if your existing version is pinned.
-
-To enable `.travis.yml` renovation, you must:
-
-1. Enable Travis renovation explicitly by setting the following Renovate configuration: `"travis": { "enabled": true }`
-1. Optionally, configure a support policy (As documented below)
-
-When Renovate processes your project's repository it will look for the files listed above and submit a single pull request that upgrades all Node.js versions simultaneously.
-
-## Configuring Support Policy
-
-Renovate supports a [`supportPolicy`](./configuration-options.md#supportpolicy) option that can be passed the following values and associated versions (current as of June 2021):
-
-**Default:** `lts`
-
-| supportPolicy | versions   | description                                              |
```

Comment:
Does this mean we don't need to sync our file that has the Node.js schedule in JSON form?

Reply:
I think we still use it for node *versioning*, e.g. to know if a release is stable or not